tests/Aspire.Hosting.Milvus.Tests/MilvusFunctionalTests.cs

```diff
@@ -0,0 +1,368 @@
+        }
+
+        var builder1 = CreateDistributedApplicationBuilder();
+        builder1.Configuration["Parameters:apikey"] = MilvusToken;
+        var apiKey1 = builder1.AddParameter("apikey");
+        var milvus1 = builder1.AddMilvus("milvus", apiKey1).WithDataBindMount(bindMountPath);
+
+        string collectionName = "book";
+
+        using (var app = builder1.Build())
+        {
+            await app.StartAsync();
+
+            var hb = Host.CreateApplicationBuilder();
+
+            hb.Configuration.AddInMemoryCollection(new Dictionary<string, string?>
+            {
+                [$"ConnectionStrings:{milvus1.Resource.Name}"] = await milvus1.Resource.ConnectionStringExpression.GetValueAsync(CancellationToken.None)
+            });
+
+            hb.AddMilvusClient(milvus1.Resource.Name);
+
+            using (var host = hb.Build())
+            {
+                await host.StartAsync();
+
+                var milvusClient = host.Services.GetRequiredService<MilvusClient>();
+                var collection = await milvusClient.CreateCollectionAsync(
+                       collectionName,
+                       new[] {
+                FieldSchema.Create<long>("book_id", isPrimaryKey:true),
+                FieldSchema.Create<long>("word_count"),
+                FieldSchema.CreateVarchar("book_name", 256),
+                FieldSchema.CreateFloatVector("book_intro", 2)
+                       });
+            }
+
+            await app.StopAsync();
+        }
+
+        var builder2 = CreateDistributedApplicationBuilder();
+        builder2.Configuration["Parameters:apikey"] = MilvusToken;
+        var apiKey2 = builder2.AddParameter("apikey");
+        var milvus2 = builder2.AddMilvus("milvus2", apiKey2).WithDataBindMount(bindMountPath);
+
+        using (var app = builder2.Build())
+        {
+            await app.StartAsync();
+
+            var hb = Host.CreateApplicationBuilder();
+
+            hb.Configuration.AddInMemoryCollection(new Dictionary<string, string?>
+            {
+                [$"ConnectionStrings:{milvus2.Resource.Name}"] = await milvus2.Resource.ConnectionStringExpression.GetValueAsync(CancellationToken.None)
+            });
+
+            hb.AddMilvusClient(milvus2.Resource.Name);
+
+            using (var host = hb.Build())
+            {
+                await host.StartAsync();
+
+                var milvusClient = host.Services.GetRequiredService<MilvusClient>();
+
+                var collections = await milvusClient.ListCollectionsAsync();
+
+                Assert.Single(collections, c => c.Name == collectionName);
+            }
+
+            await app.StopAsync();
+        }
+
+        try
+        {
+            File.Delete(bindMountPath);
+        }
+        catch
+        {
+            // Don't fail test if we can't clean the temporary folder
+        }
+    }
+
+    [Fact]
+    [RequiresDocker]
+    public async Task PersistenceIsDisabledByDefault()
```

Comment:
I don't think we need this test for every resource. I think this can be removed for Milvus.